automation/utilities/rediswq.py

```diff
@@ -20,22 +21,37 @@ class RedisWQ(object):
     https://kubernetes.io/docs/tasks/job/fine-parallel-processing-work-queue
     """
 
-    def __init__(self, name, **redis_kwargs):
-        """The default connection parameters are:
+    def __init__(self, name, max_retries=3, **redis_kwargs):
```

Comment:
Note: [Scrapy has a default of 2](https://docs.scrapy.org/en/latest/topics/downloader-middleware.html#retry-times)

Reply:
Done.